FrontEnd/src/routes/ClientRouter.jsx

```diff
@@ -1,42 +1,42 @@
-import { SignUpPage } from '../SignUp/pages/SignUpPage';
-import { SignUpModalPage } from '../SignUp/pages/SignUpModalPage';
-import { ResendActivationPage } from '../SignUp/pages/ResendActivationPage';
-import { ActivationProfilePage } from '../SignUp/pages/ActivateProfilePage';
-import { SendEmailRestorePasswordPage } from '../RestorePassword/pages/SendEmailRestorePasswordPage';
-import { RestorePasswordPage } from '../RestorePassword/pages/RestorePasswordPage';
-import { RestorePasswordModalPage } from '../RestorePassword/pages/RestorePasswordModalPage';
-import { RestorePasswordSuccessPage } from '../RestorePassword/pages/RestorePasswordSuccessPage';
-import { RestorePasswordFailedPage } from '../RestorePassword/pages/RestorePasswordFailedPage';
-import { ModerationModal } from '../moderation/ModerationModal';
-import ScrollToTopButton from '../ScrollToTopButton/ScrollToTopButton';
-import TermsAndConditions from '../terms-and-conditions-app/terms_conditions/TermsAndConditionsComponent';
-import { useAuth } from '../../hooks';
-import { Search } from '../SearchPage/Search';
-import Contact from '../Contact/Contact';
-import ErrorPage404 from '../errorPages/ErrorPage404';
+import AuthorizationPage from '../components/Authorization/AuthorizationPage';
+import CookiesPolicyComponent from '../pages/CookiesPolicyPage/CookiesPolicyComponent';
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import Loader from '../components/Loader/Loader';
+import MainPage from '../pages/LandingPage/MainPage';
+import PrivacyPolicy from '../pages/PrivacyPolicyPage/PrivacyPolicyPage';
+import ProfileDetailPage from '../pages/ProfileDetail/ProfileDetailPage';
+import ProfileListPage from '../pages/ProfileList/ProfileListPage';
+import ProfilePage from '../pages/ProfilePage/ProfilePage';
+import { SignUpPage } from '../pages/BasicPage/SignUp/SignUpPage';
+import { SignUpModalPage } from '../pages/BasicPage/SignUp/SignUpModalPage';
+import { ResendActivationPage } from '../pages/BasicPage/SignUp/ResendActivationPage';
+import { ActivationProfilePage } from '../pages/BasicPage/SignUp/ActivateProfilePage';
+import { SendEmailRestorePasswordPage } from '../components/RestorePassword/Pages/SendEmailRestorePasswordPage';
+import { RestorePasswordPage } from '../components/RestorePassword/Pages/RestorePasswordPage';
+import { RestorePasswordModalPage } from '../components/RestorePassword/Pages/RestorePasswordModalPage';
+import { RestorePasswordSuccessPage } from '../components/RestorePassword/Pages/RestorePasswordSuccessPage';
+import { RestorePasswordFailedPage } from '../components/RestorePassword/Pages/RestorePasswordFailedPage';
+import { ModerationModal } from '../components/Moderation/ModerationModal';
+import ScrollToTopButton from '../components/ScrollToTopButton/ScrollToTopButton';
+import TermsAndConditions from '../components/TermsAndConditionsApp/terms_conditions/TermsAndConditionsComponent';
```

Comment:
Since they are separate routes, it makes sense to move `RestorePassword` to the pages.